HW10/task2_fibonacci.py

```diff
@@ -0,0 +1,9 @@
+def fibonacci(n):
+    if n == 0 or n == 1:
+        return n
+    else:
+        return (fibonacci(n-1)+fibonacci(n-2))
```

Comment:
nit: more readable format
```suggestion
        return fibonacci(n-1) + fibonacci(n-2)
```